docs/modules/ROOT/pages/erc1155-pausable.adoc

```diff
@@ -0,0 +1,129 @@
+= ERC-1155 Pausable
+
+ERC1155 token with pausable token transfers, minting, and burning.
```

Comment:
Not sure about the convention here, which format should we use for contract names in adocs:
- plain text: `ERC1155`
- plain text with dash: `ERC-1155`
- link to docs: `xref:erc1155.adoc[ERC-1155]`

Reply:
Made link to docs.